src/hpr_routing.erl

```diff
@@ -18,7 +18,10 @@ init() ->
     ok = throttle:setup(?HOTSPOT_THROTTLE, HotspotRateLimit, per_second),
     ok.
 
--spec handle_packet(Packet :: hpr_packet_up:packet(), HandlerPid :: pid()) -> ok | {error, any()}.
+-spec handle_packet(
+    Packet :: hpr_packet_up:packet(),
+    HandlerPid :: grpcbox_stream:t()
```

Comment:
@michaeldjeffrey 
This type is not a pid. Below, an Error is sent to it. That can't work. For some reason, dialyzer does not complain about it.